ubuntu_touch/manifest.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "kiwix",
-  "description": "Kiwix : offline Wikipedia reader",
+  "description": "Kiwix is an offline reader for online content like Wikipedia, Project Gutenberg, or TED Talks.",
```

Comment:
The use of "online" is a bit confusing. How about:

> Kiwix is an offline reader for content from Wikipedia, Project Gutenberg, TED Talks, Wikivoyage, Stackexchange, etc.

Reply:
True. Thanks.
Changes pushed.